### DISCUSSION TITLE
How to see all nodes metrics of a cluster in a single page - kind cluster multinode

### DISCUSSION THREAD

**Discussion by Garahk** [netdata/netdata#13412] [Q&A] [answered]
Hello, I have multinode kind cluster, one control-plane and two workers. Currently taking advantage of injecting the dashboards in an angular frontend.

The cluster looks like this when I log into http://box:19999

![image](https://user-images.githubusercontent.com/99432183/180049352-c7c03fc5-dbd3-45d9-aaa9-74d3a0cd8b70.png)

Parent on high level, and then the childs and k8s.state collector.

When I log into http://box:19999/api/v1/allmetrics, naturally I will only see the parent's metrics, which don't include any of the childs' metrics or k8s.state collector. 

Based on the documentation from netdata.cloud, what needs to be used for custom dashboards is the script:

<script type="text/javascript" src="http://box:19999/dashboard.js"></script>

However as seen on the picture above, each of the metrics is on a different route, i.e.:
Parent: http://box:19999
control plane: http://box:19999/kind-control-plane
worker : http://box:19999/kind-worker
worker2 : http://box:19999/kind-worker2

Considering that on this deployment, the workers are virtual nodes, is there any way to have all the metrics concentrated on a single point of access?

**Comment by ilyam8** [comment, marked as answer]
Hi, @Garahk. I know a way - using [prometheus_all_hosts](https://learn.netdata.cloud/docs/agent/exporting/prometheus#streaming-data-from-upstream-hosts) format.

> http://box:19999/api/v1/allmetrics?format=prometheus_all_hosts


@vlvkobal are there any other options?

**Reply by Garahk** [reply]
@ilyam8 That is great!, I had worked with the API, but I didn't remember this one, the `prometheus_all_hosts` format, thank you for sharing.

@vlvkobal , just out of curiosity if there's other option, it would be good to test it out as well

**Reply by vlvkobal** [reply]
I've rechecked the source. There are no other options available.
